migration/family_trees.py

```diff
@@ -90,23 +90,22 @@ def save(self, data, name):
         data['tree_file_id'] = self.meta['file_id']
         self.onsave(data, 'persons')
 
-    def flatten(self, nodes, full=False):
+    def flatten(self, node_or_nodes, full=False):
```

Comment:
I refactored the function a bit to make it clearer that it allows either single or multiple nodes, before it was a bit unclear (to me..)